tests/bluetooth/host/keys/bt_keys_get_addr/src/test_suite_full_list_invalid_values.c

```diff
@@ -41,6 +47,16 @@ static void bt_conn_foreach_custom_fake_data_ref_null(int type, bt_conn_foreach_
 	func(&conn, NULL);
 }
 
+/* Setup test variables */
+static void every_test_setup(void *f)
```

Comment:
this seems identical to the one in main.c.
I think you can remove the `static` and declare it as an `extern`. If you did this because the `ZTEST_SUITE` macro has a limitation, then it's ok.

Reply:
I'm not sure if the test framework has such limitation, but I have a concern here.

- It's just a coincidence here that those functions match, but I prefer to keep the setup function local to the same source file that has the test case definition as part of making each test suite and its test cases independent from the others

Reply:
That's a fine argument (y)